public/angular/app_angular.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Created by Baby_Moico on 30/05/2015.
+ */
```

Comment:
I think that this file is part of your POC of learning angularJS, and I did a different way tho manage the angular app. For this reason I think that this file is useless for our propouse.